### PR TITLE
Tests separated to unit,integration and functional folders

### PR DESCRIPTION
**Pull request by akolod**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -137,6 +137,8 @@
                         <includes>
                             <include>**/*IT.java</include>
                             <include>**/*IT.groovy</include>
+                            <include>**/*FT.java</include>
```
Comment (Felipe444) [inline pom.xml:140]: do we need the rule for .java?

File: pom.xml [hunk 2]
```diff
@@ -102,7 +102,7 @@
                     <configuration>
                         <testSources>
                             <testSource>
-                                <directory>${project.basedir}/src/test/groovy</directory>
+                                <directory>${project.basedir}/src/test</directory>
                                 <includes>
                                     <include>**/*.groovy</include>
                                 </includes>
```
Comment (Felipe444) [inline pom.xml:108]: Instead of "match all" rule we can add "match UT" and we will cover all scenarios.

**Review by Felipe444 — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (akolod) [reply pom.xml:140]: I checked and we need this rule, even if all test are in groovy

File: pom.xml (hunk 2 above)
Comment (akolod) [reply pom.xml:108]: we need to specify all tests in gmavenplus-plugin, even if tests are separated

**Pull request merged**